src/renderer/register/components/AccountView/ImportView/MnemonicWord/index.js

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames';
```

Comment:
can't we reuse this as well for the mnemonic overview, when import is not selected? This also has a class "MnemonicWord" if I remember correctly

Reply:
I don't see why 🙈 this is styling for an input form